cmd/kube-apiserver/app/options/options.go

```diff
@@ -169,6 +171,8 @@ func (s *APIServer) AddFlags(fs *pflag.FlagSet) {
 	fs.StringVar(&s.ServiceAccountKeyFile, "service-account-key-file", s.ServiceAccountKeyFile, "File containing PEM-encoded x509 RSA private or public key, used to verify ServiceAccount tokens. If unspecified, --tls-private-key-file is used.")
 	fs.BoolVar(&s.ServiceAccountLookup, "service-account-lookup", s.ServiceAccountLookup, "If true, validate ServiceAccount tokens exist in etcd as part of authentication.")
 	fs.StringVar(&s.KeystoneURL, "experimental-keystone-url", s.KeystoneURL, "If passed, activates the keystone authentication plugin")
+        fs.StringVar(&s.KeystoneDomainID, "experimental-keystone-domainid", s.KeystoneDomainID, "DomainID for keystone authentication")
```

Comment:
All these options may go away, waiting for #25624 to be merged. Need more work here after that PR merged.